politeiawww/cmswww.go

```diff
@@ -865,6 +887,9 @@ func (p *politeiawww) setCMSWWWRoutes() {
 	p.addRoute(http.MethodGet, cms.APIRoute,
 		cms.RouteProposalOwner, p.handleProposalOwner,
 		permissionLogin)
+	p.addRoute(http.MethodGet, cms.APIRoute,
+		www.RouteTokenInventory, p.handlePassThroughTokenInventory,
+		permissionPublic)
```

Comment:
Maybe set this to permissionLogin . This would avoid the public hitting this heavy end point by mistake. 
```suggestion
		permissionLogin)
```